examples/8-monitoring/main.go

```diff
@@ -99,7 +101,7 @@ func process(ctx goka.Context, msg interface{}) {
 
 	u.Clicks++
 	ctx.SetValue(u)
-	fmt.Printf("[proc] key: %s clicks: %d, msg: %v\n", ctx.Key(), u.Clicks, msg)
+	// fmt.Printf("[proc] key: %s clicks: %d, msg: %v\n", ctx.Key(), u.Clicks, msg)
```

Comment:
don't you want to remove this line?

Reply:
nah, I'd rather actually comment it in again so we get the print. It's an example after all, so it does not really matter if the print is here or not.

Reply:
commented it in again.